tests/test_main.py

```diff
@@ -39,6 +39,7 @@ def test_car_wash_station(cars, wash_station, total_cost):
     income = wash_station.serve_cars(cars)
     assert income == total_cost, f"Income should equal to {total_cost}"
 
+
```

Comment:
Remove a change from this file from the pull request.

Reply:
No, you don't need any changes in the tests. For now, you have added a blank line; please rollback all changes in this file